File: app/src/main/java/com/pedrosoares/cotacaoapp/data/entity/CoinsResponse.java
package com.pedrosoares.cotacaoapp.data.entity;

import com.google.gson.annotations.SerializedName;

public class CoinsResponse{

	@SerializedName("AUD")
	private AUD aUD;

	@SerializedName("BTC")
	private BTC bTC;

	@SerializedName("CHF")
	private CHF cHF;

	@SerializedName("ARS")
	private ARS aRS;

	@SerializedName("JPY")
	private JPY jPY;

	@SerializedName("EUR")
	private EUR eUR;

	@SerializedName("GBP")
	private GBP gBP;

	@SerializedName("USD")
	private USD uSD;

	@SerializedName("CAD")
	private CAD cAD;

	@SerializedName("USDT")
	private USDT uSDT;

	@SerializedName("LTC")
	private LTC lTC;

	@SerializedName("XRP")
	private XRP xrp;

	@SerializedName("ETH")
	private ETH eth;


	public BTC getBTC(){
		return bTC;
	}

	public ARS getARS(){
		return aRS;
	}

	public EUR getEUR(){
		return eUR;
	}

	public GBP getGBP(){
		return gBP;
	}

	public USD getUSD(){
		return uSD;
	}

	public LTC getLTC(){
		return lTC;
	}

	public XRP getXrp() {
		return xrp;
	}

	public ETH getEth() {
		return eth;
	}
}

File: app/src/main/java/com/pedrosoares/cotacaoapp/data/entity/CoinsResponse.kt
package com.pedrosoares.cotacaoapp.data.entity

import com.google.gson.annotations.SerializedName

data class CoinsResponse (

        @SerializedName("AUD")
        var aud: AUD? = null,

        @SerializedName("BTC")
        var btc: BTC? = null,

        @SerializedName("CHF")
        var chf: CHF? = null,

        @SerializedName("ARS")
        var ars: ARS? = null,

        @SerializedName("JPY")
        var jpy: JPY? = null,

        @SerializedName("EUR")
        var eur: EUR? = null,

        @SerializedName("GBP")
        var gbp: GBP? = null,

        @SerializedName("USD")
        var usd: USD? = null,

        @SerializedName("CAD")
        var cad: CAD? = null,

        @SerializedName("USDT")
        var usdt: USDT? = null,

        @SerializedName("LTC")
        var ltc: LTC? = null,

        @SerializedName("XRP")
        var xrp: XRP? = null,

        @SerializedName("ETH")
        var eth: ETH? = null
)